### DISCUSSION TITLE
Archive link

### DISCUSSION THREAD

**Discussion by Mark-Macaulay** [MarimerLLC/csla#2220] [Questions]
Any chance there is an updated address to this content? (Running Side-by-side CSLA versions)
http://www.lhotka.net/cslanet/faq/CslaVersions.ashx

**Comment by rockfordlhotka** [comment]
The physical environment that was hosting the old (very old) lhotka.net server is going away, or at least won't allow my server anymore. I'm picking up a thumb drive with the vhd later this week, and can start to port important pages to the new GitHub-hosted lhotka.net. I don't plan to port everything, because I don't see the value in most of it, but I'll add this to the list.

It might be a while though, because I'm on holiday all next week and plan to actually unplug for the most part.

**Reply by Mark-Macaulay** [reply]
Thank you! I appreciate you even entertaining such a request. Your CSLA work is remarkable, you deserve all the holiday you can get.
